build.sh

```diff
@@ -225,6 +225,10 @@ while [[ $# -gt 0 ]]; do
         NO_JIT="-DNO_JIT_SH=1"
         ;;
 
+    --intl-icu)
```

Comment:
well, I would keep `--with-intl` anyways. Not everyone needs that and it consumes more memory + bigger binary size.